File: iniciante/C++/1064-Positives_and_Average.cpp
#include <iostream>
#include <iomanip>

using namespace std;
 
int main() {
 
    float num,i=0,pos=0,media=0,cont=0;
  
    for (i=0;i<6;i++) {
        cin >> num;
        if (num > 0) {
            pos += num;
            cont += 1;
        } 
    }
    cout << cont << " valores positivos\n";
    
    media = pos/cont;

    cout << fixed << setprecision(1) << media << "\n";
    return 0;
}

File: iniciante/Dart/1064-Positives_and_Average.dart
import 'dart:io';

void main() {
  int cont = 0;
  double sum = 0;
  for (int i = 0; i < 6; i++) {
    // Remove the ! before submitting the solution...

    var Num = double.parse(stdin.readLineSync()!);
    if (Num > 0) {
      cont += 1;
      sum += Num;
    }
  }

  double avg = sum / cont;
  print("${cont} valores positivos");
  print("${avg.toStringAsFixed(1)}");
}
